src/documents.jl

```diff
@@ -688,6 +688,93 @@ function lt_remotepair(r1::RemoteRepository, r2::RemoteRepository)
     return length(r1.root) > length(r2.root)
 end
 
+
+src_to_uuid = nothing
+
+function get_src_to_uuid()
+    global src_to_uuid
+    if src_to_uuid !== nothing
+        return src_to_uuid
+    end
+    src_to_uuid = Dict()
+
+    for (uuid, dep) in pairs(Pkg.dependencies())
+        src_to_uuid[dep.source] = uuid
+    end
+    return src_to_uuid
+end
+
+function get_pkginfo(uuid)
+    for registry in reachable_registries()
+        for (registry_uuid, pkg) in registry.pkgs
+            if registry_uuid == uuid
+                return registry_info(pkg)
+            end
+        end
+    end
+end
+
+function uuid_to_repo(uuid)
+    env = EnvCache()
+    package_entry = manifest_info(env.manifest, uuid)
+    pkginfo = get_pkginfo(uuid)
```

Comment:
These `Pkg` functions.. are they public API? I suspect not? Or, minimally, they're probably not documented? I am okay with relying on them but, if they're not clearly documented, we should add some tests that explicitly test their existence and the interface we expect them to have. Something like: https://github.com/JuliaDocs/DocStringExtensions.jl/blob/a970c7d7f8c8aa89a3cdae9b4da3bdb49fb12add/test/tests.jl#L22-L68